stable/airflow/README.md

```diff
@@ -322,6 +322,38 @@ To be able to expose metrics to prometheus you need install a plugin, this can b
 This exposes dag and task based metrics from Airflow.
 For service monitor configuration see the generic [Helm chart Configuration](#helm-chart-configuration).
 
+## Additional manifests 
+
+It is possible to add additional manifests into a deployment, to extend the chart. One of the reason is to have the secrets used for postgres-connection and redis-connection deployed by the chart itself.
+
+```yaml
+extraManifests: 
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: {{ template "airflow.fullname" . }}-postgres
+      labels:
+        app: {{ template "airflow.fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    type: Opaque
+    data:
+      postgres-password: "dGVzdAo="
```

Comment:
it's pretty nasty to hardcode password like this. There is no reason to use a secret in this case. I invite you to reword this feature not recommending users to deploy there passwords using manifest.

Reply:
I changed the example for the addition of a Cloud Armour config for example on gke.